AEPAssurance/Source/AssuranceSession.swift

```diff
@@ -40,9 +40,22 @@ class AssuranceSession {
     /// indicates if Assurance SDK can start forwarding events to the session. This flag is set when a command `startForwarding` is received from the socket.
     var canStartForwarding: Bool = false
 
-    /// true indicates Assurance SDK has timeout and shutdown after non-reception of deep link URL because of which it  has cleared all the queued initial SDK events from memory.
+    /// true indicates Assurance SDK has timeout and shutdown after non-reception of deep link URL because of which it has cleared all the queued initial SDK events from memory.
     var didClearBootEvent: Bool = false
 
+    /// Boolean flag indicating whether to process and queue the SDK events listened from the wildcard listener.
+    /// This flag is set to false on the following occasions
+    ///  1. When the Assurance extension automatically shuts down on non arrival of assurance deeplink after the 5 second timeout.
+    ///  2. When the Assurance session is disconnected by the user.
+    /// This flag is turned back on when
+    ///  1. Assurance extension is reconnected to an new Assurance session
```

Comment:
nit: you can change this to be a single line (instead of a list) as it seems like there is only one condition.